### ISSUE TITLE
Add "about" page

### ISSUE THREAD

**Issue by braaar** [braaar/jam-assistant#26]
I think an about page would be nice. 

```tsx
<Heading> About </Heading>
<Paragraph> Blabla bla </Paragraph>
```
For now we can just copy the text from the readme or something.

I think it could be useful to have a text that explains briefly how the page works, but ideally everything should be fairly self explanatory if we design it right.

We should add a route for it when we get the routing setup (see #10). I think `/about` is a nice path

**Comment by DipayanP007** [comment]
Hey @braaar I want to do this

**Comment by braaar** [comment]
> Hey @braaar I want to do this

Sweet! You can just add a new react component (functional component using Typescript as is the convention for this project) and add it to the `src` folder. We can figure out the routing part later once we add navigation features.

**Comment by braaar** [comment]
fixed in #27
